app/controllers/users_controller.rb

```diff
@@ -0,0 +1,65 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :edit, :update, :destroy]
+  before_action :require_login, :except => [:new, :create]
+  before_action :require_current_user, :only => [:edit, :update, :destroy]
+
+  def index
+    @users = User.all
```

Comment:
I'm curious as to what you are getting an index of users for? Is there a page that displays all users registered to the site?